build.gradle

```diff
@@ -3,10 +3,10 @@
 
 buildscript {
     ext {
-        support_version = '28.0.0'
+        support_version = '1.0.0'
```

Comment:
The name 'support_version' doesn't seem right anymore. 'androidx_version' or any other good naming.

Reply:
This is fixed, thanks